src/carousel.tsx

```diff
@@ -198,8 +198,8 @@ const BaseCarousel: React.FC<BaseCarouselProps> = ({
     const desktopContainerType = useDesktopContainerType();
     const itemsPerPageConfig = normalizeItemsPerPage(desktopContainerType || 'large', itemsPerPage);
 
-    const {isDesktopOrBigger, isMobile} = useScreenSize();
-    const mobileOrTabletItemsPerPage = isMobile ? itemsPerPageConfig.mobile : itemsPerPageConfig.tablet;
+    const {isDesktopOrBigger, isTablet} = useScreenSize();
+    const mobileOrTabletItemsPerPage = isTablet ? itemsPerPageConfig.tablet : itemsPerPageConfig.mobile;
```

Comment:
This is required to improve a bad behaviour happened in renders when the provider is not initialized, that was causing the carousel behave like in tablet, its better to behave like mobile in those cases (as the usage of tablets is lot less than mobile).